src/models/appointmentServiceViewModel.js

```diff
@@ -30,7 +30,15 @@ Bahmni.Appointments.AppointmentServiceViewModel = (function () {
 
     Service.createFromResponse = function (serviceDetails) {
         var getDateTime = function (time) {
-            return time ? new Date("January 01, 1970 " + time) : undefined;
+            if (!time) {
+                return undefined;
+            }
```

Comment:
Setting the date with "January 01, 1970" would cause issues with daylight saving time. Use the current date instead so that the returned time is ok whether we're currently on daylight saving time or not.